packages/engine/README.md

```diff
@@ -344,4 +397,4 @@ The packages utilize a [communication implementation](./src/simulation/comms) to
 
 The [DataStore](./src/datastore) is the backend responsible for keeping the data between the simulation run main loops and language runners in sync. It encapsulates logic surrounding read/write access, as well as low-level shared memory representation.
 
-[hCore]: https://core.hash.ai?utm_medium=organic&utm_source=github_readme_engine
+[hcore]: https://core.hash.ai?utm_medium=organic&utm_source=github_readme_engine
```

Comment:
This label is normalised by Prettier (no way to configure). Prettier still keeps `[hCore]` (because it’s visible), but also normalises `[visible text][hcore]`.